blocks/columns/columns.js

```diff
@@ -3,6 +3,8 @@ import { div, ul, li } from '../../scripts/dom-helpers.js';
 export default function decorate(block) {
   const cols = [...block.firstElementChild.children];
   block.classList.add(`columns-${cols.length}-cols`);
+  const anchorContainer = document.createElement('div');
+  anchorContainer.className = 'anchor-container'; // Add a class for styling if needed
```

Comment:
Class should be added by author. Not dynamically like this. Also write columns logic if within a feature flag by checking if that class exists. This way current columns code is not overridden.

Reply:
I agree, i dont see any reason of customizing JS here. Just use variation and implement the rules in CSS